xml/System.ServiceModel/EndpointAddress.xml

```diff
@@ -1447,7 +1448,6 @@
       <Docs>
         <param name="addressingVersion">The <see cref="T:System.ServiceModel.Channels.AddressingVersion" /> of the current endpoint address.</param>
         <param name="writer">The <see cref="T:System.Xml.XmlDictionaryWriter" /> to which the endpoint address is saved.</param>
-        <param name="localName">The local name for the XML node to contain the endpoint address.</param>
```

Comment:
same here - param still being deleted

Reply:
This issue will be resolved with this PR:
https://github.com/docascode/tripled/pull/1